### ISSUE TITLE
Update Project Profile: Remove Danny Do from Website Team

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#6113]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
The Hackforla.org Website project page needs to be updated so that the project information is up to date. 

### Action Items
- [x] Observe the existing syntax of the [front matter block](https://jekyllrb.com/docs/front-matter/) inside of the `_projects/website.md` file and find the `leadership` variable.
- [x] Delete the following lines from the file:
```
  - name: Danny Do
    role: Merge Team
    links:
      slack: 'https://hackforla.slack.com/team/U048G8S2YAK'
      github: 'https://github.com/Adastros'
    picture: https://avatars.githubusercontent.com/Adastros
```
- [ ] Verify the changes by viewing the [Hackforla.org Website project page](https://www.hackforla.org/projects/website) on your local environment and include a screenshot with your pull request

### Resources
- [website.md](https://github.com/hackforla/website/blob/gh-pages/_projects/website.md)
- [Jekyll Docs on front matter](https://jekyllrb.com/docs/front-matter/)

**Comment by thisisdientran** [comment]
Sure I will make it done within today.

Sorry I have 2 tests in this week making me a bit trouble in balance the
work.

Best regards,
Tuan Dien Tran
Tel: 6268786378
Linkedin: linkedin.com/in/trantuandien/

**Comment by LRenDO** [comment]
Hi @thisisdientran! I am taking you off of this issue because you are also assigned on #5710. We ask that folks not assign themselves to issues until their previous issue has been merged or closed. Once #5710 is complete you can assign yourself to another issue. Thank you for being so eager to contribute!

**Comment by zkamenov** [comment]
Availability for this week: 
Sunday 20:00-23:00 PST
MWF  19:00-23:00 PST
TR 20:00-23:00 PST
Saturday 14:00-18:00 PST

My estimated ETA for completing this issue: This Sunday at 22:00
